core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -253,6 +253,18 @@ export class RteHelperModalComponent {
         );
         return;
       }
+    } else if (this.componentId === this.COMPONENT_ID_TABS) {
+      // Value[0] corresponds to tab Contents.
+      for (let tab = 0; tab < value[0].length; tab++) {
+        if (value[0][tab].title === '' || value[0][tab].content === '') {
+          this.updateRteErrorMessage(
+            'Please ensure that each title and content are filled'
```

Comment:
Try to use a more specific message (e.g. "Please ensure that the title of tab 2 is filled in") so that the creator knows what exactly to do without hunting around.

Reply:
Changes made, thank you !